packages/flame_texturepacker/lib/flame_texturepacker.dart

```diff
@@ -9,9 +9,20 @@ export 'package:flame_texturepacker/src/texture_packer_sprite.dart';
 extension TexturepackerLoader on Game {
   /// Loads the specified pack file.
   /// Uses the parent directory of the pack file to find the page images.
+  @Deprecated('Use [fromAssets] or [fromStorage] instead')
   Future<TexturePackerAtlas> fromAtlas(
     String assetsPath, {
     bool fromStorage = false,
   }) async =>
       TexturePackerAtlas.load(assetsPath, fromStorage: fromStorage);
+
+  /// Loads the specified pack file from assets
+  /// Uses the parent directory of the pack file to find the page images.
+  Future<TexturePackerAtlas> fromAssets(String assetsPath) async =>
```

Comment:
These new names are a lot more likely to collide with future extensions, it would be good if we can name them so that it makes it clear that they are regarding the fetching an atlas, maybe `atlasFromAssets` and `atlasFromStorage`?

Reply:
I forgot this was an extension on Game. I updated the names to the ones suggested. Thanks